Doc/reference/import.rst

```diff
@@ -982,9 +982,9 @@ __main__.__spec__
 Depending on how :mod:`__main__` is initialized, ``__main__.__spec__``
 gets set appropriately or to ``None``.
 
-When Python is started with the :option:`-m` option, ``__spec__`` is set
-to the module spec of the corresponding module or package. ``__spec__`` is
-also populated when the ``__main__`` module is loaded as part of executing a
+When Python is started with the :option:`-m` option, ``__main__.__spec__`` is set
+to the module spec of the corresponding module or package. ``__main__.__spec__`` is
```

Comment:
This change does not improve the doc.  The doc explains how the `__spec__` module attribute is set, the attribute in the module namespace.  The name (one of the names) of the module is not relevant here.